### ISSUE TITLE
How to update job status after qualified on web-frontend

### ISSUE THREAD

**Issue by JK-97** [opencv/cvat#1704]
### My actions before raising this issue
- [☑] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [☑] Searched [past issues](/issues)
- [☑] searched frontend source code in cvat-ui
- [☑] tried all button in web-frontend

when i annotated  image and save , the job's status did not change to COMPLETE. 
Not in accordance with normal use logic , and no other button which can change job status.
### Expected Behaviour
N/A

### Current Behaviour
N/A

### Possible Solution
N/A
### Steps to Reproduce (for bugs)
N/A

### Context
N/A

### Your Environment

- Deploy environment : development
- Git hash commit (`git log -1`): 972e2f2ab40dd51badfaea499ec771a360722555
- Operating System and version (e.g. Linux, Windows, MacOS): linux

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

**Comment by martin-etchart** [comment]
If you are using the default (new) UI just click the "Info" in the top right:
![image](https://user-images.githubusercontent.com/1056375/84607043-33b90880-ae81-11ea-846e-9460b38f818a.png)
And change status:
![image](https://user-images.githubusercontent.com/1056375/84607049-4e8b7d00-ae81-11ea-8a74-56f27f853c73.png)

**Comment by bsekachev** [comment]
@martin-etchart Thx.
@JK-97 was your question answered?

**Comment by mirantel** [comment]
We have found this only after few weeks. 
Every time we need to change status we, went from new UI to Old just to click "Completed".
Info Icon confused. You don't expect find there changing job status.

**Comment by beasteers** [comment]
Hmm, I just pulled the cvat docker images yesterday and I don't see this option in the UI. Did it get moved? I can't find any info on it other than this issue and the other issues pointing to this one.

![image](https://user-images.githubusercontent.com/6741720/133839840-5200ad0c-25d9-4551-8bfd-5037d1f9452f.png)


It'd be great if next to the save button there was a `Save & Complete` button.

I have about 300 videos in here and it'd be great if I could keep track of which ones I've completed so far :p
